src/ui/utils/devtools-toolbox.js

```diff
@@ -59,6 +59,9 @@ export class DevToolsToolbox {
     await panel.open();
 
     this.panels[name] = panel;
+    store.dispatch(actions.setSelectedPanel(name));
+    store.dispatch(actions.setInitializedPanels(name));
```

Comment:
We have to keep a record of initializedPanels in the store. Without this, the toolbox doesn't re-render because it's not aware that a new panel has been initialized.